backend/main.py

```diff
@@ -52,9 +51,9 @@ def run_spotihue(lights: List[str]) -> None:
         track_info = spotihue.retrieve_current_track_information()
         track_album_artwork_url = track_info["track_album_artwork_url"]
 
-        redis_client.hmset(
-            "current_track_information",
-            track_info,
+        redis_client.hset(
```

Comment:
What's the difference between hmset and hset again?

Reply:
the data type that's stored should be the same - HMSET is being deprecated https://redis.io/commands/hmset/ and can be replaced with HSET! their return values are difference (a string vs an integer), but we're not checking the return value of this setting command currently, so we should be good in terms of the GET-ing